### PR TITLE
Turn on v2 telemetry for http by default

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

File: operator/data/profiles/default.yaml [hunk 1]
```diff
@@ -356,7 +356,7 @@ spec:
     telemetry:
       enabled: true
```
Comment (elfinhe) [inline operator/data/profiles/default.yaml]: I tried this yesterday, seems like telemetry v1 (mixer) also be installed by this telemetry.enabled=true. Is this what we want to keep?

Comment (howardjohn) [reply operator/data/profiles/default.yaml]: Yes. We want v2 on, which will install the EnvoyFilters and set some flag like `disableHttpMetrics`. We stll have mixer for TCP telemetry (for now)

Comment (elfinhe) [reply operator/data/profiles/default.yaml]: got it, just a double check. 

LGTM.

**Comment by howardjohn** [comment]
@mandarjog does it make sense to get this merged in now and https://github.com/istio/istio/pull/20258 since that one is blocked by other PRs?

**Comment by mandarjog** [comment]
this pr only touches tests. Did you also mean to update values file or am I missing something ?

**Comment by howardjohn** [comment]
> this pr only touches tests. Did you also mean to update values file or am I missing something ?


I did.. it must have got lost in a rebase. Ill send a fix

**Comment by howardjohn** [comment]
@mandarjog I added the real changes back, not sure how that happened.

**Review by elfinhe — APPROVED** [review]

**Pull request closed without merging**